packages/client/hmi-client/src/utils/logger.ts

```diff
@@ -84,18 +58,9 @@ class Logger {
 		try {
 			throw new Error();
 		} catch (error: any) {
-			const stack = error.stack.split('\n');
-			const callerLine = stack[3];
-			return callerLine;
+			return error.stack.split('\n')[3];
```

Comment:
Did anyone ever check if this magic-number-3 is valid or not?

Reply:
```
```
try { throw new Error("hello world"); } catch (error) { console.log(error.stack.split("\n")[3]); }

>  undefined
```